fastlane/Fastfile

```diff
@@ -99,31 +136,43 @@ in xcode user defined build configuration and just reflected in `Info.plist` dur
 
     desc "Read current version number from project configuration."
     current_version_version = read_project_property(
-      build_configuration: build_configurations[:testflight],
+      build_configuration: build_configurations[build_configuration],
       build_setting: 'CURRENT_VERSION_NUMBER'
     )
 
-    desc "Read current build number from project configuration."
-    current_build_version = read_project_property(
-      build_configuration: build_configurations[:testflight],
-      build_setting: 'CURRENT_BUILD_NUMBER'
+    desc "Read current build number from TestFlight."
+    current_build_version = latest_testflight_build_number(
+      app_identifier: bundle_identifier,
+      version: current_version_version,
+      initial_build_number: first_build
     )
 
     desc "Get the type of release desired"
+    desc "If there's only one bump_type allowed it will use that one"
+    desc "If there isn't a bump_type specified and there are many allowed bump_types, it will ask the user"
+
+    is_first_deploy = current_version_version == first_version
     bump_type = options[:bump_type]
-    while not release_bump_types.include? bump_type do
-      bump_type = UI.input "Choose the `bump_type` representing the type of deploy. It can be any of %s" % release_bump_types.to_s
+
+    if allowed_bump_types.count == 1
+      bump_type = is_first_deploy ? all_bump_types.last : allowed_bump_types.first
+    else
+      while not allowed_bump_types.include? bump_type do
+        bump_type = UI.input "Choose the `bump_type` representing the type of deploy. It can be any of %s" % allowed_bump_types.to_s
+      end
     end
 
-    if current_version_version == first_version && bump_type != release_bump_types.last
-      UI.important "This seems to be your first deploy. It must be done as %s." % release_bump_types.last
-      if (UI.input "Do you want to make it as %s? y/N" % release_bump_types.last) == "y"
-        bump_type = release_bump_types.last
+    if is_first_deploy && bump_type != all_bump_types.last
+      UI.important "This seems to be your first deploy. It must be done as %s." % all_bump_types.last
+      if (UI.input "Do you want to make it as %s? y/N" % all_bump_types.last) == "y"
+        bump_type = allowed_bump_types.last
       else
-        UI.user_error! "The first depoy must be done as %s. Please try again later." % release_bump_types.last
+        UI.user_error! "The first depoy must be done as %s. Please try again later." % all_bump_types.last
       end
     end
 
+    print "\tWill release app increasing bump type: `%s`\n" % bump_type
```

Comment:
Use [UI](https://github.com/fastlane/fastlane/blob/master/fastlane/docs/UI.md) instead of `print`. You can use `UI.message` for this.